modules/KalturaSupport/components/related/related.js

```diff
@@ -266,12 +266,11 @@
 			return this.getEntriesFromPlaylistId( '_KDP_CTXPL', callback, true);
 		},
 		isValidResult: function( data ){
-			// Check if we got error
+			// Check if we got error and if entry list is empty disable the button
 			if( !data || data.length === 0	||( data.code && data.message )	){
 				var errMsg = data.message ? data.message : 'No related items were found.';
 				this.log('Error getting related items: ' + errMsg );
-				this.updateTooltip(gM('mwe-embedplayer-related-errMsg'));
-				this.onDisable();
+				this.getBtn().hide();
```

Comment:
This will not happen as you well know, with the related if you select the "Related to entries" option it works with tags but if there are no tags it presents a list from the sphinx DB. If you select the "Playlist ID" or "Entries list" then it will always display that list for the entire playlist.

The only way for the related to not display entries is if you put an invalid entry id in the "Entries list".